lib/fix-me.js

```diff
@@ -92,3 +91,7 @@ FixMe.prototype.printIssue = function(fileName, lineNum, matchedString) {
   var issueString = JSON.stringify(issue)+"\0";
   console.log(issueString);
 }
+
+FixMe.prototype.formatPath = function(path) {
+  return path.replace(/^\/code\//, '');
+}
```

Comment:
Separated path formatting into its own method for clarity in the code and crisper tests.